### PR TITLE
[WEEKLY RELEASE] HotShot - rc-0.5.79 

### PR DESCRIPTION
**Pull request by dailinsubjam**
Closes #<ISSUE_NUMBER>

### CODE REVIEW

File: Cargo.toml
Comment (tbro) [inline Cargo.toml:64]: Looks good. Only question: should we move these from `branch` to `tag` before we merge?

Comment (dailinsubjam) [reply Cargo.toml:64]: Yes! I'll update them after weekly release PR are merged in other repos.

**Review by tbro — APPROVED** [review]
LGTM

**Comment by alxiong** [comment]
let's try to fix the Slow Test. Something is clogging it. I understand it's annoying to wait for 40min to merge the PR. 
We should try not to break the CI and investigate asap.

**Pull request merged**
